plat/arm/common/aarch64/arm_common.c

```diff
@@ -150,6 +151,7 @@ void arm_configure_sys_timer(void)
 
 	reg_val = (1 << CNTNSAR_NS_SHIFT(PLAT_ARM_NSTIMER_FRAME_ID));
 	mmio_write_32(ARM_SYS_TIMCTL_BASE + CNTNSAR, reg_val);
+#endif
```

Comment:
I can't see anywhere in your other patches where you're using this function, so is this change still needed?